openmdao/core/system.py

```diff
@@ -1843,11 +1859,14 @@ def _get_static_coloring(self):
         static = info.static
         if static is _STD_COLORING_FNAME or isinstance(static, str):
             if static is _STD_COLORING_FNAME:
-                fname = self.get_coloring_fname()
+                fname = self.get_coloring_fname(mode='input')
             else:
                 fname = static
-            print("%s: loading coloring from file %s" % (self.msginfo, fname))
+            print(f"{self.msginfo}: loading coloring from file {fname}")
             info.coloring = coloring = Coloring.load(fname)
+
+            self._save_coloring(coloring)
```

Comment:
Why didn't we save a coloring here before?

Reply:
Previously coloring files were in `$PWD/coloring_files`.  Now that we have an output directory, my intent was to save all the coloring files associated with an execution to the output directory, even if they came from another location (i.e. "input" coloring files).  This basically just says "save off the coloring being used."